tests/_support/AcceptanceTester.php

```diff
@@ -20,11 +20,21 @@
  * @SuppressWarnings(PHPMD)
  */
 class AcceptanceTester extends \Codeception\Actor {
-  use _generated\AcceptanceTesterActions;
+  use _generated\AcceptanceTesterActions {
+    waitForElement            as _waitForElement;
+    waitForElementChange      as _waitForElementChange;
+    waitForElementClickable   as _waitForElementClickable;
+    waitForElementNotVisible  as _waitForElementNotVisible;
+    waitForElementVisible     as _waitForElementVisible;
+    waitForJS                 as _waitForJS;
+    waitForText               as _waitForText;
+  }
```

Comment:
I think we don't use this type of aligned indentation in any other code, do we? Anyway, it's not a good coding style in terms of diffs - adding or removing a single line may cause changes in all others.

Reply:
I was doing this for better readability for myself while developing, but I agree, it's not what we use in our coding style. Fixed.